src/reducers/Entities/NameNoteReducer.js

```diff
@@ -34,26 +34,15 @@ export const NameNoteReducer = (state = initialState(), action) => {
       };
     }
 
-    case NAME_NOTE_ACTIONS.SAVE_EDITING: {
+    case NAME_NOTE_ACTIONS.SAVE_NEW: {
       const { editingById } = state;
       const { payload } = action;
       const { nameNote } = payload;
       const { id } = nameNote;
 
       const newEditingById = { ...editingById, [id]: nameNote };
 
-      return { ...state, editingById: newEditingById, editingId: '' };
-    }
-
-    case NAME_NOTE_ACTIONS.SAVE_NEW: {
-      const { creatingById } = state;
-      const { payload } = action;
-      const { nameNote } = payload;
-      const { id } = nameNote;
-
-      const newCreatingById = { ...creatingById, [id]: nameNote };
-
-      return { ...state, creatingById: newCreatingById };
+      return { ...state, editingById: newEditingById, editingId: id };
```

Comment:
This will be fixed up in #3671 for now though.. the selector was looking for `editingById` and not finding it, of course